agent/util-scripts/pbench-tool-meister-stop

```diff
@@ -45,8 +44,35 @@ def wait_for_pid(pid):
         time.sleep(0.1)
 
 
+class RedisServer(BaseServer):
+    """RedisServer - an encapsulation of the handling of the Redis server
+    specification.
+    """
+
+    def_port = def_redis_port
+    bad_port_ret_code = 1
+    bad_host_ret_code = 1
+    name = "Redis server"
+
+    def __init__(self, spec: str, benchmark_run_dir: Path, def_host_name: str):
+        super().__init__(spec, def_host_name)
+        try:
+            self.pid_file = (
+                benchmark_run_dir / "tm" / f"redis_{def_redis_port:d}.pid"
+            ).resolve(strict=True)
+        except FileNotFoundError:
+            self.pid_file = None
+
+    def locally_managed(self):
+        return True if self.pid_file is not None else False
```

Comment:
Why not just `return self.pid_file is not None`??

(You could just `return self.pid_file` but I do like the idea of converting the file path to a boolean here...)

Reply:
Ah, yeah, good question.  I'll fix.

Reply:
Extending @dbutenhof's suggestion:
```
    def locally_managed(self):
        return bool(self.pid_file)
```

:-)

Reply:
Because this would return `True` for any `Path` object, even if that object is not associated with a path.

Reply:
`bool(self.pid_file)` is roughly equivalent to `self.pid_file not in [0, False, None, "", (), [], {}]`, so it's actually a superset of `self.pid_file is not None`.

That is, if `self.pid_file` is a `Path` object (or anything else which is not `None`), your code will return `True` as well, just like mine does.

There's only a difference if `self.pid_file` somehow evaluates to a "Falsey" value other than `None`.